### PR TITLE
Disabled chat input in replays

### LINKED ISSUE #11743: Chat textbox should be disabled in replays
It's currently possible to type and send messages, which are then eaten by the game.  Disabling the text field would be an easy polish win.

### PR DESCRIPTION
**Pull request by abc013**
Fixes #11743.

### CODE REVIEW

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameChatLogic.cs [hunk 1]
```diff
@@ -130,6 +130,8 @@ public IngameChatLogic(Widget widget, OrderManager orderManager, World world, Mo
 			Game.BeforeGameStart += UnregisterEvents;
 
 			CloseChat();
+			if (world.IsReplay)
+				chatText.IsDisabled = () => true;
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameChatLogic.cs]: Can be simplified:

```csharp
chatText.IsDisabled = () => world.IsReplay;
```

**Comment by abc013** [comment]
Updated.

**Comment by Mailaender** [comment]
:+1:

**Comment by pchote** [comment]
Looks good to me, thanks! :+1:

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/4886cb4536f216ba6a18b874be166333a6c82db8

**Pull request merged**